requirements/uninstall-requirements.txt

```diff
@@ -7,3 +7,4 @@ pycrypto
 restkit
 http-parser
 zeep
+cython
```

Comment:
Thought of this after merging: does jsonobject need cython to be installed to run? Will this remove it (and cause things to break)? I guess since tests passed it should not be a problem.

Reply:
jsonobject doesn't need cython to run - it is only needed for compilation, which happens before publishing to pypi.